modules/cats-effect-io/src/main/scala/io/janstenpickle/trace4cats/cats/effect/package.scala

```diff
@@ -0,0 +1,37 @@
+package io.janstenpickle.trace4cats.cats
+
+import cats.effect.{IO, IOLocal}
+import io.janstenpickle.trace4cats.Span
+import io.janstenpickle.trace4cats.base.context.{Local, Provide}
+import io.janstenpickle.trace4cats.base.optics.Lens
+import io.janstenpickle.trace4cats.inject.Trace
+
+package object effect {
```

Comment:
@mrdziuban I simplified this file as I could. Still need time to check how these instances are usable